tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,9 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(vendor.inventory) == 3
+    assert item not in vendor.inventory 
+    assert result == False 
```

Comment:
Nice work! Glad to see you wrote a couple more assertions in addition to checking that result was false to make this test more robust and also check the length of the vendor's inventory didn't change unexpectedly.